constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/utils/SpringStopEngine.java

```diff
@@ -80,14 +80,14 @@ public void springConfig(float currentPos,
     }
 
     @Override
-    public float getVelocity(float t) {
+    public float getVelocity(float x) {
         return (float) mV;
     }
 
     @Override
-    public float getInterpolation(float time) {
-        compute(time - mLastTime);
-        mLastTime = time;
+    public float getInterpolation(float v) {
+        compute(v - mLastTime);
+        mLastTime = v;
         return (float) (mPos);
     }
```

Comment:
Let's instead change the names in the StopEngine interface
```Java
float getVelocity(float time)

float getInterpolation(float time)
```

Make sure the name matches in the implementations.

Reply:
This would be an interface change

Reply:
Actually no it would not be
Change it.